libethdrivers/src/plat/imx6/ocotp_ctrl.c

```diff
@@ -211,10 +210,16 @@ uint64_t ocotp_get_mac(struct ocotp *ocotp, unsigned int id)
     uint64_t mac = 0;
 
     switch (id) {
+#if defined(CONFIG_PLAT_IMX8MM_EVK) || defined(CONFIG_IMX8MQ_EVK)
+    case 0:
+        mac = ((uint64_t)((uint16_t)regs->res37[4]) << 32) | regs->res37[0];
```

Comment:
Please give the registers a name in the definition, reading a MAC from reserved registers is a bit odd.